.travis.yml

```diff
@@ -8,8 +8,4 @@ install:
   - echo Test > destination.txt
   - docker build . --tag="nwisweb_tableau_dockerinstance"
   - docker run --name nwisweb_data_connector_docker nwisweb_tableau_dockerinstance
-  - dockerinstanceid=\$( docker ps -aqf "name=nwisweb_data_connector_docker" )
-  - docker cp  "\${dockerinstanceid}\${pathtemplate}" $WORKSPACE
-  - docker cp  "\${dockerinstanceid}:usr/local/bin/nwisweb-tableau-data-connector/coverage/lcov.info" $WORKSPACE
   - cat ./coverage/lcov.info | coveralls
```

Comment:
This fails to send info to Coveralls.  Something needs to be taken out of the Docker container for this to work.